api/routes/graphs.py

```
@@ -225,3 +227,51 @@ async def delete_graph(request: Request, graph_id: str):
+                status_code=403
+            )
+
+        logging.info(
+            "Received request to update user rules, content length: %d", len(data.user_rules)
+        )
+        full_graph_id = _graph_name(request.state.user_id, graph_id)
+        await set_user_rules(full_graph_id, data.user_rules)
+        logging.info("User rules updated successfully")
+        return JSONResponse(content={"success": True, "user_rules": data.user_rules})
+    except GraphNotFoundError:
+        logging.error("Graph not found")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)